bundle/render_templates.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+set -x
+
+declare MANIFESTS_DIR
+declare METADATA_DIR
+declare CERT_MANAGER_OPERATOR_IMAGE
+declare CERT_MANAGER_WEBHOOK_IMAGE
+declare CERT_MANAGER_CA_INJECTOR_IMAGE
+declare CERT_MANAGER_CONTROLLER_IMAGE
+declare CERT_MANAGER_ACMESOLVER_IMAGE
+declare KUBE_RBAC_PROXY_IMAGE
+
+CSV_FILE_NAME="cert-manager-operator.clusterserviceversion.yaml"
+ANNOTATIONS_FILE_NAME="annotations.yaml"
+
+update_csv_manifest()
+{
+  CSV_FILE="${MANIFESTS_DIR}/${CSV_FILE_NAME}"
+  if [[ ! -f ${CSV_FILE} ]]; then
+    echo "[$(date)] -- ERROR -- operator csv file \"${CSV_FILE}\" does not exist"
+    exit 1
+  fi
+
+  ## replace cert-manager operand related images
+  sed -i "s#quay.io/jetstack/cert-manager-webhook.*#${CERT_MANAGER_WEBHOOK_IMAGE}#g" ${CSV_FILE}
+  sed -i "s#quay.io/jetstack/cert-manager-controller.*#${CERT_MANAGER_CONTROLLER_IMAGE}#g" ${CSV_FILE}
+  sed -i "s#quay.io/jetstack/cert-manager-cainjector.*#${CERT_MANAGER_CA_INJECTOR_IMAGE}#g" ${CSV_FILE}
+  sed -i "s#quay.io/jetstack/cert-manager-acmesolver.*#${CERT_MANAGER_ACMESOLVER_IMAGE}#g" ${CSV_FILE}
+
+  ## replace kube-rbac-proxy image
+  sed -i "s#gcr.io/kubebuilder/kube-rbac-proxy.*#${KUBE_RBAC_PROXY_IMAGE}#g" ${CSV_FILE}
+
+  ## replace cert-manager-operator image
+  sed -i "s#openshift.io/cert-manager-operator.*#${CERT_MANAGER_OPERATOR_IMAGE}#g" ${CSV_FILE}
+
+  ## add annotations
+  yq e -i ".metadata.annotations.createdAt=\"$(date -u +'%Y-%m-%dT%H:%M:%S')\"" "${CSV_FILE}"
+  yq e -i '.metadata.annotations."features.operators.openshift.io/cnf"="false"' "${CSV_FILE}"
+  yq e -i '.metadata.annotations."features.operators.openshift.io/cni"="false"' "${CSV_FILE}"
+  yq e -i '.metadata.annotations."features.operators.openshift.io/csi"="false"' "${CSV_FILE}"
```

Comment:
any specific reason why we need to add these feature annotations in here and not directly in the ClusterServiceVersion like the other existing feature annotations?
https://github.com/openshift/cert-manager-operator/blob/6ab8b4c05df78dfb95212079de84c9dc256046df/bundle/manifests/cert-manager-operator.clusterserviceversion.yaml#L215-L221

Also, are the `cnf`, `csi`, `cni` required required until recently? If so, would prefer to add directly in CSV rather than here IMO.

Reply:
`operator-sdk` tool does not add these annotations and the `make bundle` step will remove these annotations. These annotations are mandated in the enterprise policy.

Reply:
Any specific known reason why `make bundle` removes those feature annotations? 

Or, could we just try with adding onto `bundle/manifests/cert-manager-operator.clusterserviceversion.yaml`.
```diff
+ features.operators.openshift.io/cnf"="false"
+ features.operators.openshift.io/cni"="false"
+ features.operators.openshift.io/csi"="false"
```

Lest does a bump help, if not we can use the original script lines as-is, for a last resort maybe.

Reply:
comment incorporated.